<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-O0M08K0" failures="1" tests="1" name="Test1" time="49.179" errors="0" timestamp="2025-03-25T20:14:21 IST">
  <testcase classname="contactTest.ToCreateContactTest" name="toCreateContact_001" time="27.552">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at contactTest.ToCreateContactTest.toCreateContact_001(ToCreateContactTest.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- toCreateContact_001 -->
</testsuite> <!-- Test1 -->
